src/qt_gui/translations/pl_PL.ts

```diff
@@ -1,491 +1,493 @@
+        <translation>To oprogramowanie nie służy do grania w gry pochodzące z nielegalnego źródła.</translation>
+    </message>
+</context>
+<context>
+    <name>ElfViewer</name>
+    <message>
+        <location filename="../elf_viewer.cpp" line="45"/>
+        <source>Open Folder</source>
+        <translation>Otwórz folder</translation>
+    </message>
+</context>
+<context>
+    <name>GameInfoClass</name>
+    <message>
+        <location filename="../game_info.cpp" line="26"/>
+        <source>Loading game list, please wait :3</source>
+        <translation>Ładowanie listy gier, proszę poczekaj :3</translation>
+    </message>
+    <message>
+        <location filename="../game_info.cpp" line="26"/>
+        <source>Cancel</source>
+        <translation>Anuluj</translation>
+    </message>
+    <message>
+        <location filename="../game_info.cpp" line="27"/>
+        <source>Loading...</source>
+        <translation>Ładowanie...</translation>
+    </message>
+</context>
+<context>
+    <name>GameInstallDialog</name>
+    <message>
+        <location filename="../game_install_dialog.cpp" line="24"/>
+        <source>shadPS4 - Choose directory</source>
+        <translation>shadPS4 - Wybierz katalog</translation>
+    </message>
+    <message>
+        <location filename="../game_install_dialog.cpp" line="31"/>
+        <source>Directory to install games</source>
+        <translation>Katalog do instalacji gier</translation>
+    </message>
+    <message>
+        <location filename="../game_install_dialog.cpp" line="50"/>
+        <source>Browse</source>
+        <translation>Przeglądaj</translation>
+    </message>
+    <message>
+        <location filename="../game_install_dialog.cpp" line="74"/>
+        <source>Error</source>
+        <translation>Błąd</translation>
+    </message>
+    <message>
+        <location filename="../game_install_dialog.cpp" line="75"/>
+        <source>The value for location to install games is not valid.</source>
+        <translation>Podana ścieżka do instalacji gier nie jest prawidłowa.</translation>
+    </message>
+</context>
+<context>
+    <name>GuiContextMenus</name>
+    <message>
+        <location filename="../gui_context_menus.h" line="39"/>
+        <source>Create Shortcut</source>
+        <translation>Utwórz skrót</translation>
+    </message>
+    <message>
+        <location filename="../gui_context_menus.h" line="40"/>
+        <source>Open Game Folder</source>
+        <translation>Otwórz katalog gry</translation>
+    </message>
+    <message>
+        <location filename="../gui_context_menus.h" line="41"/>
+        <source>SFO Viewer</source>
+        <translation>Menedżer plików SFO</translation>
+    </message>
+    <message>
+        <location filename="../gui_context_menus.h" line="42"/>
+        <source>Trophy Viewer</source>
+        <translation>Menedżer trofeuów</translation>
```

Comment:
```suggestion
        <translation>Menedżer trofeów</translation>
```
"trofeów" is genitive, plural form for this noun, right? Some ref in polish: https://wsjp.pl/haslo/podglad/26899/trofeum/2584240/sportowe
Nit: If this is just a preview/browser of collected/avaliable trophies, perhaps "Przeglądarka trofeów" may be more appropiate, even thought its direct english equivalent will be "trophy browser".

Reply:
I've picked `explorer` for for all, as it's more fitting.